examples/contracts/aggregator_consumer/Makefile

```diff
@@ -1,12 +1,53 @@
-# TODO: these helper commands may be useful for the deploy script(s)
-#
-# ACCOUNT_NAME="test-acct"
-# 
-# create-account:
-# 	@sncast account create --name $(ACCOUNT_NAME)
-# 
-# deploy-account:
-# 	@sncast account deploy --name $(ACCOUNT_NAME)
-
-run-script:
-	@cd ./scripts && sncast script run $(NAME)
+# The starknet-devnet-rs container version - this version 
+# supports rpc v0.7 which is needed for snfoundry v0.20.1
+CONTAINER_VERSION="7743a089a33beb75d7012e4aa24745bee8ae0d71"
+
+# The name of the account that will be used for snfoundry
+# scripts + transactions
+ACCOUNT_NAME="test-acct"
+
+devnet:
+	@printf "\nStarting a local starknet devnet docker container:\n\n" \
+		&& bash ../../../ops/scripts/devnet-hardhat-down.sh \
+		&& CONTAINER_VERSION="$(CONTAINER_VERSION)" bash ../../../ops/scripts/devnet-hardhat.sh \
+		&& printf "\n\nIf you intend to use a local starknet devnet container to run these examples,\nadd \"url = http://localhost:5050/rpc\" in the snfoundry.toml file\n\n"
+
+create-account:
+	@printf "\nCreating an account named \"$(ACCOUNT_NAME)\"...\n\n" \
+		&& sncast account create --name $(ACCOUNT_NAME) \
+	  && printf "\nYour accounts:\n\n" \
+		&& cat ~/.starknet_accounts/starknet_open_zeppelin_accounts.json \
+		&& printf "\n\nYou can fund your account here: https://sepolia.starkgate.starknet.io\n"
+
+add-account:
+	@printf "\nAdding an account named \"$(ACCOUNT_NAME)\"...\n\n" \
+		&& sncast \
+			--url "http://127.0.0.1:5050/rpc" \
+			account add \
+			--name $(ACCOUNT_NAME) \
+			--address "0x4b3f4ba8c00a02b66142a4b1dd41a4dfab4f92650922a3280977b0f03c75ee1" \
+			--private-key "0x57b2f8431c772e647712ae93cc616638" \
+	  && printf "\nYour accounts:\n\n" \
+		&& cat ~/.starknet_accounts/starknet_open_zeppelin_accounts.json
+
+view-accounts:
+	@cat ~/.starknet_accounts/starknet_open_zeppelin_accounts.json
+
+deploy-account:
+	@sncast account deploy --name $(ACCOUNT_NAME)
+
+delete-account:
+	@sncast account delete --name $(ACCOUNT_NAME)
+
+tc-deploy:
+	@cd ./scripts && sncast --account "$(ACCOUNT_NAME)" script run deploy_test_consumer
+
+tc-set-latest-round:
+	@cd ./scripts && sncast --account "$(ACCOUNT_NAME)" script run set_latest_round 
```

Comment:
You can define `account = "$ACCOUNT_NAME"` in the toml config under the default profile, then we don't need to specify `--account` everywhere

Reply:
Done!

- In `snfoundry.toml` the account name is now configurable through an environment variable
- The `Makefile` is now the only place where the account names for devnet and testnet are defined - it is not hardcoded in any other place